packages/react-native/src/theme/tokens/index.ts

```diff
@@ -0,0 +1,53 @@
+import { components } from './components';
```

Comment:
This file can just be located in _theme/tokens.ts_. We can remove the _tokens_ directory entirely